### PR TITLE
Add spec and parser for alternatives_display_python

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending approval of Spec

File: insights/parsers/alternatives.py
Comment (bfahr) [inline insights/parsers/alternatives.py:165]: @JoySnow the `shared[...]` method is deprecated and no longer valid, please update your documentation and add testing for your docstrings.  See the [documentation](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for more information.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/alternatives.py
Comment (JoySnow) [reply insights/parsers/alternatives.py]: Hi @bfahr, Thanks for the info. The docstrings part is enhanced. Please kindly review.

**Review by bfahr — APPROVED** [review]

**Pull request merged**
